app/get_proc_arg.go

```diff
@@ -0,0 +1,231 @@
+package app
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+
+	"github.com/rancher/longhorn-manager/types"
+)
+
+const (
+	DetectPodMaxPolls = 120
+
+	ArgFlexvolumePluginDir = "volume-plugin-dir"
+	ArgKubeletRootDir      = "root-dir"
+
+	DefaultFlexvolumeDir  = "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/"
+	DefaultKubeletRootDir = "/var/lib/kubelet"
+
+	DetectFlexvolumeName     = "discover-flexvolume-dir"
+	DetectKubeletRootDirName = "discover-kubelet-root-dir"
+
+	DetectKubeletArgScript = `
+    find_kubelet_proc() {
+      for proc in $(find /proc -maxdepth 1 -type d 2>/dev/null); do
+        if [ ! -f $proc/cmdline ]; then
+          continue
+        fi
+        if [[ "$(cat $proc/cmdline | tr '\000' '\n' | head -n1 | tr '/' '\n' | tail -n1)" == "kubelet" ]]; then
+          echo $proc
+        fi
+      done
+    }
+    get_kubelet_arg_dir() {
+      proc=$(find_kubelet_proc)
+      if [ "$proc" != "" ]; then
+        path=$(cat $proc/cmdline | tr '\000' '\n' | grep %s | tr '=' '\n' | tail -n1)
+        if [ "$path" == "" ]; then
+          echo "Cannot found arg root-dir in proc kubelet. Will use default root-dir."
+          echo %s
+        else
+          echo $path
+		fi
```

Comment:
This line is using tabs instead of spaces. It's inconsistent with the lines above.